testing/jormungandr-integration-tests/src/networking/p2p/connections.rs

```diff
@@ -146,7 +148,34 @@ fn gossip_interval() {
         .spawn(SpawnParams::new(CLIENT).in_memory())
         .unwrap();
 
-    utils::wait(10);
+    //check server gets the gossip sent by client every default gossip_interval
+    let last_gossip = server
+        .rest()
+        .network_stats()
+        .unwrap()
+        .last()
+        .unwrap()
+        .last_gossip_received
+        .unwrap();
+
+    utils::wait(DEFAULT_GOSSIP_INTERVAL_SECS + 1);
```

Comment:
Why `DEFAULT_GOSSIP_INTERVAL_SECS + 1` maybe just `DEFAULT_GOSSIP_INTERVAL_SECS` ?

Reply:
To make the test a bit more robust, but it will probably be fine without the +1